### PR TITLE
[PAPI-194] Explicitly refer to mining governance

### PR DESCRIPTION
**Pull request by drmathias**

### CODE REVIEW

File: src/Opdex.Platform.Infrastructure.Abstractions/Data/Models/MiningGovernances/MiningGovernanceNominationEntity.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
 using Opdex.Platform.Common.Models.UInt;
 
-namespace Opdex.Platform.Infrastructure.Abstractions.Data.Models.Governances
+namespace Opdex.Platform.Infrastructure.Abstractions.Data.Models.MiningGovernances
 {
     public class MiningGovernanceNominationEntity : AuditEntity
     {
         public ulong Id { get; set; }
-        public ulong GovernanceId { get; set; }
+        public ulong MiningGovernanceId { get; set; }
```
Comment (mrtpain) [inline src/Opdex.Platform.Infrastructure.Abstractions/Data/Models/MiningGovernances/MiningGovernanceNominationEntity.cs:8]: ahhh yikes, these will need sql changes on any Entity properties.

File: src/Opdex.Platform.Infrastructure/Data/Handlers/MiningGovernances/Nominations/SelectActiveGovernanceNominationsByIdQueryHandler.cs [hunk 2]
```diff
@@ -27,21 +27,21 @@ public class SelectActiveGovernanceNominationsByGovernanceIdQueryHandler
...
-                  {nameof(MiningGovernanceNominationEntity.GovernanceId)} = @{nameof(SqlParams.GovernanceId)}
-            LIMIT {GovernanceConstants.MaxNominations};";
+                  {nameof(MiningGovernanceNominationEntity.MiningGovernanceId)} = @{nameof(SqlParams.MiningGovernanceId)}
+            LIMIT {MiningGovernanceConstants.MaxNominations};";
 
         private readonly IDbContext _context;
         private readonly IMapper _mapper;
 
-        public SelectActiveGovernanceNominationsByGovernanceIdQueryHandler(IDbContext context, IMapper mapper)
+        public SelectActiveMiningGovernanceNominationsByIdQueryHandler(IDbContext context, IMapper mapper)
```
Comment (mrtpain) [inline src/Opdex.Platform.Infrastructure/Data/Handlers/MiningGovernances/Nominations/SelectActiveGovernanceNominationsByIdQueryHandler.cs:36]: This name and the associated query should be `ByMiningGovernanceId`. Might need this change for Retrieve etc. too

File: src/Opdex.Platform.Application/Assemblers/LiquidityPoolDtoAssembler.cs [hunk 3]
```diff
@@ -116,8 +116,8 @@ public async Task<LiquidityPoolDto> Assemble(LiquidityPool pool)
                 poolDto.Summary.Staking.SetDailyChange(previousPoolSnapshot?.Staking?.Weight ?? UInt256.Zero);
 
                 // Set Nomination Status
-                var governance = await _mediator.Send(new RetrieveMiningGovernanceByTokenIdQuery(stakingTokenDto.Id));
-                var nominations = await _mediator.Send(new RetrieveActiveGovernanceNominationsByGovernanceIdQuery(governance.Id));
+                var miningGovernance = await _mediator.Send(new RetrieveMiningGovernanceByTokenIdQuery(stakingTokenDto.Id));
+                var nominations = await _mediator.Send(new RetrieveActiveMiningGovernanceNominationsByIdQuery(miningGovernance.Id));
```
Comment (mrtpain) [inline src/Opdex.Platform.Application/Assemblers/LiquidityPoolDtoAssembler.cs:120]: Same comment `ByMiningGovernanceId` long name sucks but 🤷‍♂️

**Comment by mrtpain** [comment]
Created a PR to deploy alongside this for the UI: https://github.com/Opdex/opdex-ui/pull/65

File: src/Opdex.Platform.Infrastructure/Data/Handlers/MiningGovernances/Nominations/SelectActiveGovernanceNominationsByIdQueryHandler.cs (hunk 2 above)
Comment (drmathias) [reply src/Opdex.Platform.Infrastructure/Data/Handlers/MiningGovernances/Nominations/SelectActiveGovernanceNominationsByIdQueryHandler.cs]: Good point, nominations have an id so `ById` isn't clear

File: src/Opdex.Platform.Infrastructure.Abstractions/Data/Models/MiningGovernances/MiningGovernanceNominationEntity.cs (hunk 1 above)
Comment (drmathias) [reply src/Opdex.Platform.Infrastructure.Abstractions/Data/Models/MiningGovernances/MiningGovernanceNominationEntity.cs:8]: Updated the table names, should be ready to go once db scripts are approved

**Review by mrtpain — APPROVED** [review]

**Pull request merged**